src/main/java/software/amazon/awssdk/crt/io/ClientBootstrap.java

```diff
@@ -17,6 +17,17 @@
 
     private final CompletableFuture<Void> shutdownComplete = new CompletableFuture<>();
 
+    /**
+     * Creates a new ClientBootstrap with a default HostResolver and EventLoopGroup.
+     * Most applications will only ever need one instance of this.
+     */
+    public ClientBootstrap() throws CrtRuntimeException {
```

Comment:
I'm not sure this is a useful public API.  Consider making it private since the only one who should ever instantiate is the static GetOrCreate